docker/plugin-updates/Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+def versionTag = "v${env.BUILD_NUMBER}"
+pipeline {
+  agent {
+    ecs {
+      inheritFrom 'plugin-updates'
+    }
+  }
+
+  stages {
+    stage('Test') {
+      steps {
+        script {
+          def branch = plugin-updates-${versionTag}
+          checkout scm
+          sh 'plugin-updates'
+          sh 'git status'
+          sh "git checkout -b ${branch}"
+          sh 'git add  docker/plugins.txt'
+          sh "git config --global user.email tna-digital-archiving-jenkins@nationalarchives.gov.uk"
+          sh "git config --global user.name tna-digital-archiving-jenkins"
```

Comment:
Just a thought - I wonder if this should be in the Dockerfile for all of the Jenkins build nodes? I think it's in a few Jenkinsfiles, but we could set it on the image instead.

Reply:
Yeah makes sense, I've set it in the Dockerfile for this build and removed it from the Jenkinsfile and it's all working.

Reply:
Thanks. I've made a card in case we want to come back and do this in the other build nodes: https://national-archives.atlassian.net/browse/TDR-910